content/client-server-api/modules/content_repo.md

```diff
@@ -19,6 +19,12 @@ When serving content, the server SHOULD provide a
 `Content-Security-Policy` header. The recommended policy is
 `sandbox; default-src 'none'; script-src 'none'; plugin-types application/pdf; style-src 'unsafe-inline'; object-src 'self';`.
 
+{{% added-in v="1.4" %}}
+
+The server SHOULD additionally provide `Cross-Origin-Resource-Policy: cross-origin`
+when serving content to allow (web) clients access to APIs which interact
+with the media repository, such as `SharedArrayBuffer`.
```

Comment:
this doesn't sound right. `SharedArrayBuffer` doesn't interact with the media repo?

I think this should be something like '... allow clients which interact with the media repo to access restricted APIs such as `SharedArrayBuffer` by setting a `Cross-Origin-Embedder-Policy: require-corp` header', though I may be getting it slightly mixed up

Reply:
I think your wording is more correct though ultimately the same depending on perspective - will play with the wording a bit.